src/types/redis_string.cc

```diff
@@ -181,6 +182,48 @@ rocksdb::Status String::GetEx(engine::Context &ctx, const std::string &user_key,
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status String::DelEX(engine::Context &ctx, const std::string &user_key, DelExOption &option,
+                              std::optional<std::string> &hash_or_value, bool &deleted) {
+  std::string ns_key = AppendNamespacePrefix(user_key);
+  std::string value;
+  rocksdb::Status s = getValue(ctx, ns_key, &value);
+  if (!s.ok() || s.IsNotFound()) {
+    return s;
+  }
```

Comment:
`IsNotFound` is included by `!s.ok()`

```suggestion
  if (!s.ok()) return s;
```